app/models/enrollment.rb

```diff
@@ -0,0 +1,4 @@
+class Enrollment < ApplicationRecord
```

Comment:
Right  @lastk. We will make a `checklist` to check if someone attended or not to the event.